lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -398,7 +398,10 @@ def _call(self, method: Callable, params: ApiParams):
             raise APIAbort(e.http_status, str(e))
         except KeyError as e:
             self.logger.exception("{} problem in preprocess, missing {}", klasname, e)
-            raise APIAbort(HTTPStatus.INTERNAL_SERVER_ERROR)
+            raise APIAbort(
+                HTTPStatus.INTERNAL_SERVER_ERROR,
+                f"Missing expected backend data: {str(e)}",
```

Comment:
Should we also use the `repr` form here like in all the other changes that report the exception message in this module?

Reply:
Missed that one, although I went back and forth on the philosophical point. Should we quote the exception text, or just leave it inline?